config/passport.js

```diff
@@ -0,0 +1,62 @@
+const passport = require( 'passport' )
+const User = require( '../models/users' )
+
+const { Strategy, ExtractJwt } = require( 'passport-jwt' )
+
+const LocalStrategy = require( 'passport-local' )
+
+const localOptions = {
+  usernameField: 'email'
+}
+
+const ERROR_MESSAGE = {
+  error: 'Your login details could not be verified. Please try again.'
+}
+
+const strategy = ( email, password, done ) => {
+  User.findOne({ email }, ( error, user ) => {
+    if(error) {
+      return done( error )
+    }
+
+    if(!user) {
+      return done( null, false, ERROR_MESSAGE )
+    }
+
+    user.comparePassword( password, function( error, isMatch ) {
+      if(error) {
+        return done( error )
+      }
+
+      if(!isMatch) {
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```